imports/plugins/included/surcharges/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,286 @@
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+
+  "This defines the surcharge that you want to create"
+  surcharge: SurchargeInput!
+
+  "The shop to create this surcharge for"
+  shopId: ID!
+}
+
+"Input for the `deleteSurcharge` mutation"
+input DeleteSurchargeInput {
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+
+  "The ID of the flat rate fulfillment method you want to delete"
+  surchargeId: ID!
+
+  "The shop that owns the method"
+  shopId: ID!
+}
+
+"Input to add a surcharge message with language"
+input MessagesByLanguageInput {
+  content: String!
+  language: String!
+}
+
+"Input to add a surcharge attribute restriction"
+input SurchargeAttributeRestrictionsInput {
+  property: String
+  value: String
+  propertyType: String
+  operator: String
+}
+
+"Input to add a surcharge destination restriction"
+input SurchargeDestinationRestrictionsInput {
+  country: [String]
+  region: [String]
+  postal: [String]
+}
+
+"Defines a surcharge."
+input SurchargeInput {
+  "Method IDs to apply this surcharge to."
+  methodIds: [String]
+
+  """
+  The type of this surcharge. Allowed types are "surcharge".
+  """
+  type: SurchargeTypeEnum!
+
+  "Attribute restrictions."
+  attributes: [SurchargeAttributeRestrictionsInput]
+
+  "Destination restrictions."
+  destination: SurchargeDestinationRestrictionsInput
+
+  "Amount."
+  amount: Float!
+
+  "Messages by language."
+  messagesByLanguage: [MessagesByLanguageInput]!
+}
+
+"Input for the `updateSurcharge` mutation"
+input UpdateSurchargeInput {
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+
+  "The shop that owns the method"
+  shopId: ID!
+
+  "The ID of the flat rate fulfillment surcharge you want to update"
+  surchargeId: ID!
+
+  "The updated surcharge. Pass the whole updated surcharge object without the ID."
+  surcharge: SurchargeInput!
+}
+
+
+####
+# Queries
+####
+extend type Query {
+  "Get the full list of flat rate fulfillment method restrictions."
+  getSurcharges(
+    shopId: ID!,
+    after: ConnectionCursor,
+    before: ConnectionCursor,
+    first: ConnectionLimitInt,
+    last: ConnectionLimitInt,
+    sortOrder: SortOrder = desc,
+    sortBy: SurchargeSortByField = createdAt
+  ): SurchargeConnection!
+
+  getSurcharge(shopId: ID!, surchargeId: ID!): Surcharge
+}
+
+
+####
+# Mutations
+####
+extend type Mutation {
+  "Create a surcharge"
+  createSurcharge(input: CreateSurchargeInput!): CreateSurchargePayload!
+}
+
+extend type Mutation {
+  "Delete a flat rate fulfillment restriction"
+  deleteSurcharge(input: DeleteSurchargeInput!): DeleteSurchargePayload!
+}
+
+extend type Mutation {
+  "Update a flat rate fulfillment surcharge"
+  updateSurcharge(input: UpdateSurchargeInput!): UpdateSurchargePayload!
+}
```

Comment:
All three of these fields can be wrapped in a single `extend type Mutation` block